cluster-endpoints/src/grpc/gprc_accounts_streaming.rs

```diff
@@ -0,0 +1,215 @@
+                    Default::default(),
+                    Default::default(),
+                    Some(commitment),
+                    Default::default(),
+                    None,
+                )
+                .await?;
+
+            while let Some(message) = stream.next().await {
+                let Ok(message) = message else {
+                    continue;
+                };
+
+                let Some(update) = message.update_oneof else {
+                    continue;
+                };
+
+                match update {
+                    UpdateOneof::Account(account) => {
+                        if let Some(account_data) = account.account {
+                            let account_pk_bytes: [u8; 32] = account_data
+                                .pubkey
+                                .try_into()
+                                .expect("pubkey should be deserializable");
+                            let owner: [u8; 32] = account_data
+                                .owner
+                                .try_into()
+                                .expect("owner pubkey should be deserializable");
+                            let notification = AccountNotificationMessage {
+                                data: AccountData {
+                                    pubkey: Pubkey::new_from_array(account_pk_bytes),
+                                    account: Account {
+                                        lamports: account_data.lamports,
+                                        data: account_data.data,
+                                        owner: Pubkey::new_from_array(owner),
+                                        executable: account_data.executable,
+                                        rent_epoch: account_data.rent_epoch,
+                                    },
+                                    updated_slot: account.slot,
+                                },
+                                commitment: Commitment::Confirmed,
+                            };
+                            if account_stream_sx.send(notification).is_err() {
+                                log::error!("Account stream broken");
+                                break;
+                            }
+                        }
+                    }
+                    UpdateOneof::Ping(_) => {
+                        log::trace!("GRPC Ping");
+                    }
+                    _ => {
+                        log::trace!("unknown GRPC notification");
+                    }
+                };
+            }
+            log::error!("Grpc account subscription broken (resubscribing)");
+            tokio::time::sleep(std::time::Duration::from_secs(1)).await;
+        }
+    })
+}
+
+pub fn create_grpc_account_streaming(
+    grpc_sources: Vec<GrpcSourceConfig>,
+    accounts_filters: AccountFilters,
+) -> (AnyhowJoinHandle, AccountStream) {
+    let (processed_account_sender, processed_account_stream) =
```

Comment:
what does the word _processed_ mean?
and **if** it means something, it should be part of the method name too iMO